cmd/olympus/actions/package_downloader.go

```diff
@@ -51,7 +52,7 @@ func GetPackageDownloaderJob(s storage.Backend, e eventlog.Eventlog, w worker.Wo
 		}
 
 		// save it
-		if err := s.Save(module, version, modBytes, zipBytes, infoBytes); err != nil {
+		if err := s.Save(context.Background(), module, version, modBytes, zipBytes, infoBytes); err != nil {
```

Comment:
nit for future: let the called pass the context to `GetPackageDownloaderJob`

Reply:
I decided  to defer on this and just use context.Background() because:

- I imagine that each invocation of a download job should get its own Context.
- Adding a Context to the parameter list of `GetPackageDownloaderJob` job will only give us a context that was created at App initialization time, not per Job invocation.
- I think we'd need to change the Buffalo Worker interface to get this right.

Reply:
good points all around! ignore me then :)